staging/nvidia/charts/nvidia-dcgm-exporter/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 appVersion: "0.1.0"
 description: Nvidia GPU Metrics Exporter
 name: nvidia-dcgm-exporter
-version: 0.1.1
+version: 0.1.2
 maintainers:
-  - name: gilbert88
\ No newline at end of file
+  - name: gilbert88
```

Comment:
As @Gilbert88 isn't with the company, anymore, you might want to think about adding yourself (and/or anyone you see appropriate) here.

Reply:
Done